runtime/gc_glue_java/configure_includes/configure_linux_riscv.mk

```diff
@@ -0,0 +1,109 @@
+###############################################################################
+# Copyright (c) 2019, 2019 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+include $(CONFIG_INCL_DIR)/configure_common.mk
+
+# The riscv64 cross toolchain is used for both 32bit and 64bit
+RISCV_CROSSTOOLS_PREFIX=riscv64-unknown-linux-gnu
+
+CONFIGURE_ARGS += \
+	--enable-debug \
+	--enable-OMR_THR_THREE_TIER_LOCKING \
+	--enable-OMR_THR_YIELD_ALG \
+	--enable-OMR_THR_SPIN_WAKE_CONTROL \
+	--enable-OMRTHREAD_LIB_UNIX \
+	--enable-OMR_ARCH_RISCV \
+	--enable-OMR_ENV_LITTLE_ENDIAN \
+	--enable-OMR_PORT_CAN_RESERVE_SPECIFIC_ADDRESS \
+	--enable-OMR_GC_IDLE_HEAP_MANAGER \
+	--enable-OMR_GC_TLH_PREFETCH_FTA \
+	--enable-OMR_GC_CONCURRENT_SCAVENGER \
+	--enable-OMR_GC_ARRAYLETS
+
+ifneq (,$(findstring _riscv64, $(SPEC)))
+	CONFIGURE_ARGS += \
```

Comment:
There should be no tab characters preceding macro definitions in this file; leading tabs should only be in the script of a rule and there are no rules here.

Reply:
Already replaced with spaces as suggested.